packages/terra-clinical-item-collection/src/ItemCollection.scss

```diff
@@ -1,10 +1,27 @@
 :local {
-  .column-accessory,
-  .list-accessory {
+  .table {
+    table-layout: fixed;
+  }
+
+  .start-accessory-column {
     width: 40px;
   }
 
-  .content-accessory {
+  .content-accessory-align-top {
+    vertical-align: top !important;
```

Comment:
Why are these `!important` needed?

Reply:
For whatever reason the terra-table styles are applied over the Item Collection styles

Reply:
It looks like the table styles do have a higher specificity (1 class and 1 element). I think if you change these selectors to be `.content-accessory.content-accessory-align-top` then the specificity should be high enough with 2 classes to override the table styles without the `!important`

Reply:
That worked! Thank you!!